crates/sui-graphql-rpc/src/types/epoch.rs

```diff
@@ -19,6 +19,9 @@ use diesel::{ExpressionMethods, QueryDsl, SelectableHelper};
 use sui_indexer::models_v2::epoch::QueryableEpochInfo;
 use sui_indexer::schema_v2::epochs;
 
+/// Operation of the Sui network is temporally partitioned into non-overlapping,
+/// fixed-duration epochs.
```

Comment:
curious: fixed by time or by number of transactions or by checkpoints? I feel like I should know this already lol :sweat_smile:

Reply:
Epoch length is controlled by time, note that it's not a precise fixed duration though, so I would weaken the phrasing there -- the network aims to keep epochs roughly the same length as each other.